README.md

```diff
@@ -6,684 +6,46 @@
 
 # Mobile Center SDK for iOS
 
-## Introduction
+Mobile Center is mission control for mobile apps.
+Get faster release cycles, higher-quality apps, and the insights to build what users want.
 
-The Microsoft Mobile Center iOS SDK allows you to add Mobile Center services to your iOS application.
+The Mobile Center SDK uses a modular architecture so you can use just one or all of the following services: 
 
-1. **Analytics**: Mobile Center Analytics helps you understand user behavior and customer engagement to improve your iOS app. The SDK automatically captures session count, device properties like model, OS version etc. and pages. You can define your own custom events to measure things that matter to your business. All the information captured is available in the Mobile Center portal for you to analyze the data.
+1. **Mobile Center Analytics**: Mobile Center Analytics helps you understand user behavior and customer engagement to improve your app. The SDK automatically captures session count, device properties like model, OS version etc. and pages. You can define your own custom events to measure things that matter to your business. All the information captured is available in the Mobile Center portal for you to analyze the data.
```

Comment:
Automatic page tracking is disabled. Do we need to mention it there?
The SDK automatically captures session count, device properties like model, OS version etc. and **pages**.